demos/golang/grpc_pingpong/run_pong_on_occlum.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+occlum_pong="./occlum_pong"
+
+if [ ! -f $occlum_pong ];then
+    echo "Error: cannot stat file '$occlum_pong'"
+    echo "Please see README and build it using prepare_pong_pong.sh"
+    exit 1
+fi
+
+# Init Occlum Workspace
+rm -rf occlum_pong_instance && mkdir occlum_pong_instance
+cd occlum_pong_instance
+occlum init
+sed -i 's/256MB/2560MB/g' ./Occlum.json
+sed -i 's/32MB/320MB/g' ./Occlum.json
+sed -i 's/1MB/10MB/g' ./Occlum.json
+sed -i 's/4MB/40MB/g' ./Occlum.json
+sed -i 's/32MB/320MB/g' ./Occlum.json
+sed -i 's/80MB/960MB/g' ./Occlum.json
```

Comment:
The `jq` tool is more suitable to manipulate JSON file. Using `jq` is more readable and less error prone. Check out the example [here](https://github.com/occlum/occlum/blob/master/demos/java/run_java_on_occlum.sh#L27).

Reply:
Done